models/Discord/Commands/DiscordCommandMentorWho.js

```diff
@@ -0,0 +1,28 @@
+const DiscordCommand = require('../DiscordCommand.js');
+const StringUtils = require('../../Utils/String.js');
+
+class DiscordCommandMentorWho extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("mentorwho", "Get current mentors online.", 'mentorwho', subsystem);
+  }
+
+  onRun(message, permissions, args) {
+    var config = this.subsystem.manager.getSubsystem("Config").config;
```

Comment:
🤔

Reply:
you gotta say something more than the thinking emoji